VERSION

```diff
@@ -1,5 +1,7 @@
 v9.1.0
 
+v9.1.0 Fixes for unused aws import in DynamoDB tests
+
```

Comment:
So we were already on v9.1.0. This file should only be updated when/if you bump the version. 

Bumping to v9.1.1 is perfectly fine.

Reply:
Got it, will update the version. 

make generate wasn't working due to me using go1.20. This works with go1.16.